tests/run-unit-tests.scm

```diff
@@ -293,8 +293,8 @@
       (apply
        append
        (map
-        (lambda (f)
-          (find-files (path-expand f file-or-dir) filter))
+        (lambda (f) (if (equal? f "03-number") '()
```

Comment:
Can we put back the `03-number` unit tests now?  I see only a difference of
```
heine:~/programs/gambit/gambit/tests> ./run-unit-tests.scm 
[270|  0] 100% ████████████████ 7.6s
PASSED ALL 270 UNIT TESTS
heine:~/programs/gambit/gambit/tests> ./run-unit-tests.scm 
[310|  0] 100% ████████████████ 8.9s
PASSED ALL 310 UNIT TESTS
```
without and with the `03-number` tests.